src/components/DataTable/DataTable-story.js

```diff
@@ -1,503 +1,459 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
-
-import DataTableRow from '../DataTableRow';
-import DataTableData from '../DataTableData';
-import {
-  DataTable,
-  DataTableToolbar,
-  DataTableToolbarContent,
-  DataTableToolbarAction,
-  DataTableHead,
-  DataTableColumnHeader,
-  DataTableBody,
-  DataTableBatchActions,
-  DataTableActionList,
-  DataTableBatchAction,
-  DataTableSearch,
-  DataTableContainer,
-  DataTableSelectAll,
+import DataTable, {
+  Table,
+  TableBatchAction,
+  TableBatchActions,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableExpandHeader,
+  TableExpandRow,
+  TableExpandedRow,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableSelectAll,
+  TableSelectRow,
+  TableToolbar,
+  TableToolbarAction,
+  TableToolbarContent,
+  TableToolbarSearch,
 } from '../DataTable';
-import PaginationV2 from '../PaginationV2';
 import Button from '../Button';
-import OverflowMenu from '../OverflowMenu';
-import OverflowMenuItem from '../OverflowMenuItem';
-
-const paginationProps = {
-  pageSizes: [10, 20, 30, 40, 50],
-};
 
 const initialRows = [
   {
```

Comment:
Small thing - but until we have a more fleshed out dummy content created, do you mind re-naming the examples to at least have different values in each row (so different port numbers, different protocols, different rules, etc)? Just so the sorting and search functionality becomes more apparent :-)

Reply:
Yupp! Makes sense to me 😄  Let me add a TODO